### ISSUE TITLE
Migrations are not performed if version number starts with a "v"

### ISSUE THREAD

**Issue by multiwebinc** [wintercms/winter#466]
### Winter CMS Build

1.1

### PHP Version

7.4

### Database engine

SQLite

### Issue description

As per the documentation at https://wintercms.com/docs/plugin/updates, in the "version.yaml" file, versions are prepended with "v", however, in practice, this does not work. The migration files referenced under that version are not executed if the version begins with a "v".

### Steps to replicate

Straight from the documentation at https://wintercms.com/docs/plugin/updates:

```yaml
"v1.1.1":
    - "v1.1.1/some_upgrade_file.php"
    - "v1.1.1/some_seeding_file.php"
```

The above two scripts are not executed. However the following works correctly:

```yaml
1.1.1:
    - "v1.1.1/some_upgrade_file.php"
    - "v1.1.1/some_seeding_file.php"
```

**Comment by multiwebinc** [comment]
In `modules/system/classes/versionManager.php` > `updatePlugin()`

```php
if ($currentVersion == $databaseVersion) {
    $this->note(' - <info>Nothing to update.</info>');
    return;
}
```

Note: `'string' == 0` returns `true` prior to PHP 8 (see https://www.php.net/manual/en/types.comparisons.php). Before any migrations are performed, the version is set to `0`.

**Comment by LukeTowers** [comment]
@multiwebinc can you try: `if ($currentVersion === (string) $databaseVersion) {` instead?

**Comment by multiwebinc** [comment]
@LukeTowers Close. However, for some reason, it skips the initial version.

**Comment by multiwebinc** [comment]
Ok, I got it figured out. PR coming.

**Comment by LukeTowers** [comment]
@multiwebinc is this fixed in the 1.2 branch? See https://github.com/wintercms/winter/commit/af311f7001d9f07c802166faafaa0d5715a2dc78

**Comment by multiwebinc** [comment]
@LukeTowers I apologize for the delay in getting back to you on this. I just tested it and it appears to be working for me.
